### PR TITLE
Documentation: Update Contributing.md and storybook/README.md 

### PR DESCRIPTION
**Pull request by nikolasrieble**
## Description
Goal: Make it easier for new contributors to get an overview over current initiatives and focus.
In context: https://github.com/recharts/recharts/discussions/3727

Update Contributing.md 
- Remove explicit code style suggestion: What the linter does counts. Nobody will bother conform to rules in a file manually. If we care we should update the linter. 
- Remove mention of the demo. Very much out of date, and does not have high value. 
- Mention Storybook

Update storybook/README.md
- Update according the latest development. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,36 +2,37 @@
 
 We'd love for you to contribute to our source code and to make Recharts even better than it is today! Here are the guidelines we'd like you to follow:
 
-+ [Issues and Bugs](#issues)
-+ [Pull Requests](#pr)
-+ [Code Guide](#code)
-+ [License](#license)
+- [Issues and Bugs](#issues)
+- [Pull Requests](#pr)
+- [License](#license)
```
Comment (nikolasrieble) [inline CONTRIBUTING.md:7]:
```suggestion
- [Code Guide](#code)
- [License](#license)
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,36 +2,38 @@
...
-+ [Code Guide](#code)
-+ [License](#license)
+- [Issues and Bugs](#issues)
+- [Pull Requests](#pr)
+- [Code Guide](#code)
+- [License](#license)
+
+## Ongoing Initiatives
+
+We use Github Discussion to organise our discussion along new intiatives and organise ourselves.
```
Comment (nikolasrieble) [inline CONTRIBUTING.md:12]:
```suggestion
We use Github Discussion to organise our discussion along new intiatives and organise ourselves. Please do check the [Announcements](https://github.com/recharts/recharts/discussions/categories/announcements) section for an overview of recent initiatives. 
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -2,36 +2,38 @@
...
-
-+ Fork the repo and create your branch from `master`.
-+ If you've added code that should be **tested**, add tests!
-+ If you've changed APIs, update the [documentation](https://github.com/recharts/recharts.org) and demo.
-+ Ensure the test suite passes (`npm run test`).
-+ Make sure your code lints (`npm run lint`) - we've done our best to make sure these rules match our internal linting guidelines.
+- Search [GitHub](https://github.com/recharts/recharts/pulls) for an open or closed Pull Request that relates to your submission. You don't want to duplicate effort.
 
+- Fork the repo and create your branch from `master`.
+- If you have added functionality or changed existing functionality, be sure to add a test. Ideally a unit test for helper function, or a test that includes rendering with RTL.
```
Comment (elvezpablo) [inline CONTRIBUTING.md:33]: Could a link be added to an example test and how to run it? Sometimes it difficult to figure out how to run a test in isolation and what a good test looks like for a new project.

**Review by ckifer — APPROVED** [review]
LGTM, section on testing would be a good add as follow up or in this change

File: CONTRIBUTING.md (hunk 3 above)
Comment (nikolasrieble) [reply CONTRIBUTING.md:52]: I added a whole section on testing. Thank you, good pointer!

**Comment by nikolasrieble** [comment]
@ckifer Re-requesting review, as I have added a section on testing.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -2,36 +2,57 @@
...
 
 ### Where to Find Known Issues
 
-We will be using [GitHub Issues](https://github.com/recharts/recharts/issues) for our public bugs. We will keep a close eye on this and try to make it clear when we have an internal fix in progress. Before filing a new task, try to make sure your problem doesn't already exist.
+We will be using [GitHub Issues](https://github.com/recharts/recharts/issues) for our bugs and feature requests. We will keep a close eye on this and try to make it clear when we have an internal fix in progress. Before filing a new task, try to make sure your problem doesn't already exist.
 
 ### Reporting New Issues
 
-The best way to get your bug fixed is to provide a reduced test case. jsFiddle  provide a way to give live examples. You can fork our example in [recharts.org](http://recharts.org/) to show your case.
+The best way to get your bug fixed is to provide a reduced test case. jsFiddle provide a way to give live examples. You can fork our example in [recharts.org](http://recharts.org/) to show your case.
```
Comment (ckifer) [inline CONTRIBUTING.md:23]: Missed this last review, we should mention codesandbox instead of jsfiddle

**Review by ckifer — APPROVED** [review]
1 tiny tiny thing that can be follow up

File: CONTRIBUTING.md (hunk 4 above)
Comment (nikolasrieble) [inline CONTRIBUTING.md:23]:
```suggestion
The best way to get your bug fixed is to provide a reduced test case. codesandbox provide a way to give live examples. You can fork our example in [recharts.org](http://recharts.org/) to show your case.
```

**Pull request merged**